### PR TITLE
Aws config version

### PR DESCRIPTION
**Pull request by hunchback**
make version configurable by:

```
pipeline:
  transform:
    type: awsflowlogs
      awsflowlogs:
        version: 2
```

### CODE REVIEW

File: secadvisor/pkg/transform_test.go [hunk 1]
```diff
@@ -168,25 +156,25 @@ func Test_Transform_basic_flow(t *testing.T) {
 	f := getFlow()
 	transformer.extendGremlin.populateExtendGremlin(f, transformer)
 	secAdvFlow := transformer.Transform(f).(*SecurityAdvisorFlow)
-	assertEqual(t, version, secAdvFlow.Version)
-	assertEqualInt64(t, 0, secAdvFlow.UpdateCount)
-	assertEqual(t, "STARTED", secAdvFlow.Status)
-	assertEqualInt64(t, 1546338030000, secAdvFlow.Start)
-	assertEqualInt64(t, 1546338030000, secAdvFlow.Last)
-	assertEqual(t, "fake_node_type", secAdvFlow.NodeType)
+	core.AssertEqual(t, version, secAdvFlow.Version)
```
Comment (dubek) [inline secadvisor/pkg/transform_test.go:159]: Where is the definition of `core.AssertEqual`? I see no changes to files under `core`.

File: secadvisor/pkg/mangle_logstatus_test.go [hunk 2]
```diff
@@ -34,18 +31,8 @@ pipeline:
...
-	r := bytes.NewBuffer(content)
-	if err := cfg.ReadConfig(r); err != nil {
-		t.Fatalf("Failed to read config: %s", err)
-	}
-	return cfg
-}
-
 func getMangleLogStatus(t *testing.T) core.Mangler {
-	cfg := ConfigFromJSON(t, logStatusConfig)
+	cfg := core.ConfigFromJSON(t, logStatusConfig)
```
Comment (dubek) [inline secadvisor/pkg/mangle_logstatus_test.go:35]: where is the definition of this function in `core` ?

**Review by dubek — CHANGES_REQUESTED** [review]

File: secadvisor/pkg/mangle_logstatus_test.go (hunk 2 above)
Comment (hunchback) [reply secadvisor/pkg/mangle_logstatus_test.go:35]: added

File: secadvisor/pkg/transform_test.go (hunk 1 above)
Comment (hunchback) [reply secadvisor/pkg/transform_test.go:159]: added

File: core/utils.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+func AssertEqualInt64(t *testing.T, expected, actual int64) {
+	t.Helper()
+	if expected != actual {
+		t.Fatalf("Equal assertion failed: (expected: %v, actual: %v)", expected, actual)
+	}
+}
+
+func ConfigFromJSON(t *testing.T, content []byte) *viper.Viper {
+	cfg := viper.New()
+	cfg.SetConfigType("yaml")
```
Comment (dubek) [inline core/utils.go:43]: I think the function name should be `ConfigFromYAML` to be true to its actual implementation.

File: core/utils.go [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+	r := bytes.NewBuffer(content)
+	if err := cfg.ReadConfig(r); err != nil {
```
Comment (dubek) [inline core/utils.go:45]: Maybe use `buf` instead of `r` as the variable name?

File: core/utils.go (hunk 3 above)
Comment (hunchback) [reply core/utils.go:43]: will handle in a later PR

File: core/utils.go (hunk 4 above)
Comment (hunchback) [reply core/utils.go:45]: will handle in a later PR

**Pull request merged**